ereolen_support/ereolen_support.module

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen support.
+ */
+
+// Required to load form with fields.
+module_load_include('inc', 'ereolen_support', 'ereolen_support.form');
+
+/**
+ * Implements hook_menu().
+ */
+function ereolen_support_menu() {
+  $items['admin/config/ereolen_support'] = [
```

Comment:
Should this have been placed under `admin/config/ereolen/support` as `admin/config/ereolen` exists?

Reply:
Changed, and added to eReolen group